pages/fellows/2024/JordanAshley.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: fellow
+pagetype: fellow
+shortname: JordanAshley
+permalink: /fellows/JordanAshley.html
+fellow-name: Jordan Science Ashley
+title: Jordan Science Ashley - IRIS-HEP Fellow
+active: True
+dates:
+  start: 2024-05-20
+  end: 2024-08-11
+photo: /assets/images/team/fellows-2024/Jordan-Ashley.png
+institution: University of Tennessee, Knoxville
+e-mail: jay.ashley@me.com
+project_title: Packaging jet substructure observable tools
+project_goal: >
+    The objective of this project is to repackage the EnergyFlow and Wasserstein tools for modern PyPI and conda-forge.
```

Comment:
Can shorten this up slighlty.

```suggestion
    Repackage the EnergyFlow and Wasserstein for PyPI and conda-forge using modern build tools.
```

Reply:
I implemented the change but also removed "the" after "Repackage."